internal/worker/worker.go

```diff
@@ -101,6 +98,11 @@ func (h *ConditionTaskHandler) Handle(ctx context.Context, condition *rctypes.Co
 		return controller.ErrRetryHandler
 	}
 
+	// TODO: encrypt BMC credentials before store in prod
```

Comment:
We can't leave that as a `TODO`. Flasher should not encrypt anything on its own for any storage. It should always get credentials for the BMC from FleetDB.

Reply:
Can you kill that `TODO` line then please? Just so any folks who might be snooping on our code don't get a Wrong Idea? tyvm.